<ui version="4.0" >
 <class>GuiTransactionInfoBase</class>
 <widget class="QWidget" name="GuiTransactionInfoBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form1</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonBack" >
       <property name="text" >
        <string>&amp;Back</string>
       </property>
       <property name="shortcut" >
        <string>Alt+B</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>321</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" >
    <widget class="QFrame" name="frame3" >
     <property name="frameShape" >
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="margin" >
       <number>11</number>
      </property>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="labelType" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Box</enum>
            </property>
            <property name="text" >
             <string>Single Transaction</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel6_2" >
            <property name="text" >
             <string>&lt;b>Own Account:&lt;/b></string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame4" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Plain</enum>
            </property>
            <layout class="QGridLayout" >
             <property name="margin" >
              <number>11</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item row="2" column="0" >
              <widget class="QLabel" name="textOurBankCode1" >
               <property name="text" >
                <string>BankCode:</string>
               </property>
               <property name="wordWrap" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="0" >
              <widget class="QLabel" name="labelOurCountry1" >
               <property name="text" >
                <string>Country:</string>
               </property>
               <property name="wordWrap" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="1" >
              <widget class="QLabel" name="labelOurBankCode2" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="frameShape" >
                <enum>QFrame::Box</enum>
               </property>
               <property name="text" >
                <string/>
               </property>
               <property name="wordWrap" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="1" >
              <widget class="QLabel" name="labelOurCountry2" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="frameShape" >
                <enum>QFrame::Box</enum>
               </property>
               <property name="text" >
                <string/>
               </property>
               <property name="wordWrap" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="1" >
              <widget class="QLabel" name="labelOurAccountId2" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="frameShape" >
                <enum>QFrame::Box</enum>
               </property>
               <property name="text" >
                <string/>
               </property>
               <property name="wordWrap" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0" >
              <widget class="QLabel" name="textOurAccount1" >
               <property name="text" >
                <string>Account Id:</string>
               </property>
               <property name="wordWrap" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0" >
              <widget class="QLabel" name="labelOurName1" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text" >
                <string>Owner:</string>
               </property>
               <property name="wordWrap" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1" >
              <widget class="QLabel" name="labelOurName2" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="frameShape" >
                <enum>QFrame::Box</enum>
               </property>
               <property name="text" >
                <string/>
               </property>
               <property name="wordWrap" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel6_2_2" >
            <property name="text" >
             <string>&lt;b>Other Account:&lt;/b></string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame4_2" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Plain</enum>
            </property>
            <layout class="QGridLayout" >
             <property name="margin" >
              <number>11</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item row="2" column="0" >
              <widget class="QLabel" name="labelOtherBankCode1" >
               <property name="text" >
                <string>BankCode:</string>
               </property>
               <property name="wordWrap" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="1" >
              <widget class="QLabel" name="labelOtherBankCode2" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="frameShape" >
                <enum>QFrame::Box</enum>
               </property>
               <property name="text" >
                <string/>
               </property>
               <property name="wordWrap" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="1" >
              <widget class="QLabel" name="labelOtherCountry2" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="frameShape" >
                <enum>QFrame::Box</enum>
               </property>
               <property name="text" >
                <string/>
               </property>
               <property name="wordWrap" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1" >
              <widget class="QLabel" name="labelOtherName2" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="frameShape" >
                <enum>QFrame::Box</enum>
               </property>
               <property name="text" >
                <string/>
               </property>
               <property name="wordWrap" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0" >
              <widget class="QLabel" name="labelOtherName1" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text" >
                <string>Owner:</string>
               </property>
               <property name="wordWrap" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="1" >
              <widget class="QLabel" name="labelOtherAccountId2" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="frameShape" >
                <enum>QFrame::Box</enum>
               </property>
               <property name="text" >
                <string/>
               </property>
               <property name="wordWrap" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0" >
              <widget class="QLabel" name="labelOtherAccountId1" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text" >
                <string>Account Id:</string>
               </property>
               <property name="wordWrap" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="0" >
              <widget class="QLabel" name="labelOtherCountry1" >
               <property name="text" >
                <string>Country:</string>
               </property>
               <property name="wordWrap" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QFrame" name="frame6" >
          <property name="frameShape" >
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Plain</enum>
          </property>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>11</number>
           </property>
           <item>
            <layout class="QVBoxLayout" >
             <property name="spacing" >
              <number>6</number>
             </property>
             <property name="margin" >
              <number>0</number>
             </property>
             <item>
              <layout class="QGridLayout" >
               <property name="margin" >
                <number>0</number>
               </property>
               <property name="spacing" >
                <number>6</number>
               </property>
               <item row="1" column="1" >
                <widget class="QLabel" name="labelPrimanota2" >
                 <property name="sizePolicy" >
                  <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="frameShape" >
                  <enum>QFrame::Box</enum>
                 </property>
                 <property name="text" >
                  <string/>
                 </property>
                 <property name="wordWrap" >
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" >
                <widget class="QLabel" name="labelValue1" >
                 <property name="text" >
                  <string>Value:</string>
                 </property>
                 <property name="wordWrap" >
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="1" >
                <widget class="QLabel" name="labelValue2" >
                 <property name="sizePolicy" >
                  <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="frameShape" >
                  <enum>QFrame::Box</enum>
                 </property>
                 <property name="text" >
                  <string/>
                 </property>
                 <property name="wordWrap" >
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="0" >
                <widget class="QLabel" name="labelPrimanota1" >
                 <property name="text" >
                  <string>Primanota:</string>
                 </property>
                 <property name="wordWrap" >
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel6" >
          <property name="text" >
           <string>&lt;b>Purpose:&lt;/b></string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelPurpose" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>3</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape" >
           <enum>QFrame::Box</enum>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame23" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape" >
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>11</number>
           </property>
           <item>
            <layout class="QVBoxLayout" >
             <property name="spacing" >
              <number>6</number>
             </property>
             <property name="margin" >
              <number>0</number>
             </property>
             <item>
              <spacer>
               <property name="orientation" >
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>20</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QGridLayout" >
               <property name="margin" >
                <number>0</number>
               </property>
               <property name="spacing" >
                <number>6</number>
               </property>
               <item row="0" column="1" >
                <widget class="QLabel" name="labelFirstExecution2" >
                 <property name="sizePolicy" >
                  <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="frameShape" >
                  <enum>QFrame::Box</enum>
                 </property>
                 <property name="text" >
                  <string/>
                 </property>
                 <property name="wordWrap" >
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="2" column="1" >
                <widget class="QLabel" name="labelThree2" >
                 <property name="sizePolicy" >
                  <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="frameShape" >
                  <enum>QFrame::Box</enum>
                 </property>
                 <property name="text" >
                  <string/>
                 </property>
                 <property name="wordWrap" >
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" >
                <widget class="QLabel" name="labelFirstExecution1" >
                 <property name="text" >
                  <string>First Execution:</string>
                 </property>
                 <property name="wordWrap" >
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="1" >
                <widget class="QLabel" name="labelLastExecution2" >
                 <property name="sizePolicy" >
                  <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="frameShape" >
                  <enum>QFrame::Box</enum>
                 </property>
                 <property name="text" >
                  <string/>
                 </property>
                 <property name="wordWrap" >
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="0" >
                <widget class="QLabel" name="labelLastExecution1" >
                 <property name="text" >
                  <string>Last Execution:</string>
                 </property>
                 <property name="wordWrap" >
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="2" column="0" >
                <widget class="QLabel" name="labelThree1" >
                 <property name="text" >
                  <string>labelThree</string>
                 </property>
                 <property name="wordWrap" >
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer>
               <property name="orientation" >
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>20</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
